CONTRIBUTING.md

```diff
@@ -337,6 +337,10 @@ docker run --rm --volume=$(pwd):/srv/jekyll -it jekyll/jekyll:stable jekyll buil
 # 4. Remove the old site and copy the new one.
```

Comment:
Please improve the formatting .. It is easier to miss some step especially for `index.md` part .. may be newline after each step would be better or anything else if you think of.